style.css

```diff
@@ -528,3 +517,604 @@ textarea {
+    .project-title-1 {
+      display: none;
+    }
+
+    .desktop-header,
+    .list-items,
+    .bold-text-desktop,
+    .text-1-desktop,
+    .primary-text-desktop,
+    .ruby,
+    .project-title-desktop,
+    .project-title1-desktop {
+      display: flex;
+    }
+
+    .list-item {
+      padding-left: 32px;
+    }
+
+    .headline {
+      padding: 150px 100px 55px 100px;
+    }
+
+    .container,
+    .container-1 {
+      display: flex;
+      margin: 60px;
+      gap: 2px;
+    }
+
+    .container-1 {
+      padding: 2px;
+    }
+
+    .desktop-title {
+      width: 100%;
+    }
+
+    .card-image-desktop,
+    .card-image-desktop1 {
+      display: flex;
+      width: 100%;
+      border-radius: 8px;
+      padding: 16px;
+    }
+
+    .card-img-desktop {
+      width: 100%;
+    }
+
+    .card-image-desktop1 {
+      order: 2;
+    }
+
+    .project-title-desktop,
+    .project-title1-desktop {
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 700;
+      font-size: 32px;
+      line-height: 44px;
+      color: #172b4d;
+      width: 100%;
+      height: 36px;
+      padding-top: 30px;
+      padding-bottom: 30px;
+    }
+
+    .bold-text-desktop {
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 600;
+      font-size: 13px;
+      line-height: 16px;
+      color: #344563;
+      height: 16px;
+      margin-left: 16px;
+    }
+
+    .text-1-desktop {
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 600;
+      font-size: 13px;
+      line-height: 16px;
+      color: #7a869a;
+      height: 16px;
+    }
+
+    .primary-text-desktop {
+      width: 90%;
+      height: 79px;
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 400;
+      font-size: 15px;
+      line-height: 24px;
+      color: #344563;
+      margin: 0 16px;
+    }
+
+    .tags {
+      height: 8vh;
+      width: 100%;
+      padding-left: 6px;
+    }
+
+    .project,
+    .project1 {
+      width: 100%;
+    }
+
+    .button {
+      background-color: #fff;
+      color: #396df2;
+      padding: 12px;
+      text-align: center;
+      text-decoration: none;
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 500;
+      font-size: 17px;
+      line-height: 24px;
+      margin: 20px 16px;
+      cursor: pointer;
+      border: 1px solid #6070ff;
+      border-radius: 8px;
+    }
+
+    .ruby {
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 500;
+      font-size: 12px;
+      line-height: 16px;
+      color: #6070ff;
+      border-radius: 8px;
+      width: 110px;
+      height: 24px;
+      background: #ebebff;
+      padding: 4px 12px 4px 12px;
+      margin: 8px;
+      justify-content: center;
+    }
+
+    .base {
+      display: flex;
+    }
+
+    .form-content-container {
+      padding: 100px 250px 0 250px;
+    }
+
```

Comment:
Please make sure the `margin` background-image position, button size, and input-form size and all of your designs are the same as the Figma design in your contact form.

Your design 
![image](https://github.com/JoyTaribagshaw/My-Portfolio/assets/101523582/01d7d010-cd1e-4792-bdf8-0a9b14e284d0)
Figma design
![image](https://github.com/JoyTaribagshaw/My-Portfolio/assets/101523582/f2d797d8-e294-4588-9da7-6824f3ff803f)

Reply:
![MicroverseScreenshot4](https://github.com/JoyTaribagshaw/My-Portfolio/assets/111312788/b6d06f64-4354-40a2-a3f7-8114cdac9593)
Thank you so much for your review, I have made the necessary changes.